python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,101 @@
+import argparse, os, sys, time
+import logging
+from core.keys import keys_show
+from modules.auth.query import query_account
+from modules.bank.query import query_balances
+from utils.bank import print_balance_deductions
+from utils.txs import create_signed_txs, create_unsigned_txs, create_multi_messages
+from utils.types import account_type, num_txs_type
+
+CHAINID = os.getenv('CHAINID')
+DAEMON = os.getenv('DAEMON')
+DAEMON_HOME = os.getenv('DAEMON_HOME')
+DENOM = os.getenv('DENOM')
+HOME = os.getenv('HOME')
+
+parser = argparse.ArgumentParser(description='This program takes inputs for intializing multi message load test.')
+parser.add_argument('-s', '--sender', type = account_type, default = keys_show("account1")[1]['address'], help = 'Sender bech32 address')
+parser.add_argument('-r', '--reciever', type= account_type, default = keys_show("account2")[1]['address'], help= 'Reciever bech32 address')
+parser.add_argument('-n', '--num_txs', type = int, default = 1000, help= 'Number of transactions to be made, Min. should be 1000')
+args = parser.parse_args()
+FROM, TO, NUM_TXS = args.sender, args.reciever, int(args.num_txs)
+
+if FROM == TO:
+    sys.exit('Error: The values of arguments "TO" and "FROM" are equal make sure to set different values')
+ 
+acc1, acc2, num_msgs = FROM, TO, 30
```

Comment:
Why `num_msgs` value declared as 30?

Reply:
Yes, let's use 30 as default but should be configurable via ENV setting